### PR TITLE
Updated link to forums

### PR DESCRIPTION
**Pull request by JeffBail**
The link to forums was broken. I replaced it with a working link directly to the Unity forum.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Copyright 2011-2015 ARToolworks, Inc.
 
 [website]: http://www.artoolkit.org
 [documentation]: http://artoolkit.org/documentation/
-[forums]: http://artoolkit.org/community/
+[forums]: http://artoolkit.org/community/forums/viewforum.php?f=28&sid=5a057fb8c24ecbb4dc7c34534af91074
```
Comment (ThorstenBux) [inline README.md]: Hi Jeff,

the link includes your session id. I think better would be:
http://artoolkit.org/community/forums/viewforum.php?f=28

without the sid.
Cheers

Comment (JeffBail) [inline README.md]: Oops sorry about that. I've committed the fix.

Thanks,
Jeff

**Comment by ThorstenBux** [comment]
I think it is a good idea that it links directly to the Unity forum. 
Maybe you can change the text in the support section that it states 'forum' instead of 'forums' and fix the link to the '[bug tracker]' as well? It should point to the _Issues_ tab.

Only it you got time. Otherwise ping the conversation shortly and I'll hit merge.

**Comment by JeffBail** [comment]
Done!

**Comment by ThorstenBux** [comment]
Awesome :-)
👍

**Pull request merged**